django_scheduled_tasks/models.py

```diff
@@ -0,0 +1,43 @@
+import importlib
+from django.utils import timezone
+from django.db import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+
+
+class ScheduledTask(models.Model):
+    """
+    A record represents a single scheduled (recurring) task
+    """
+    func = models.CharField(max_length=100, unique=True)  # importable function name
+    interval_minutes = models.IntegerField(validators=[MinValueValidator(1),
+                                                       MaxValueValidator(1440)])  # 1 min to 24 hours
+    enabled = models.BooleanField(default=True)
+    last_timestamp = models.DateTimeField(null=True, blank=True)
+    last_success = models.BooleanField(null=True, blank=True)
+
+    def execute(self):
+        """ A callable function, found using our string func value """
```

Comment:
Other doc strings seem to be on new lines like below example. Not a big issue, but nice for consistency
```
"""
xxxxxx
"""
```

Reply:
For some of our projects we've started using https://github.com/pycqa/flake8-docstrings - which insists that single-line docstrings should have their quotes on a single line too. So I've started adopting that convention everywhere.